### ISSUE TITLE
Path getter can return null

### ISSUE THREAD

**Issue by zegnus** [miguelpruivo/flutter_file_picker#823]
**Describe the bug**
The [PlatformFile](https://github.com/miguelpruivo/flutter_file_picker/blob/master/lib/src/platform_file.dart#L6) object can be instantiated with a null path:
```
class PlatformFile {
  PlatformFile({
    String? path,
    required this.name,
    required this.size,
    this.bytes,
    this.readStream,
  }) : _path = path;
```

But the public method `String get path` does not allow to return null:
```
String get path {
    if (kIsWeb) {
      /// https://github.com/miguelpruivo/flutter_file_picker/issues/751
      throw '''
      On web `path` is always `null`,
      You should access `bytes` property instead,
      Read more about it [here](https://github.com/miguelpruivo/flutter_file_picker/wiki/FAQ)
      ''';
    }
    return _path!;
  }
```

where
```
    return _path!;
```

will throw an error

**Platform**
- [X] Android
- [X] iOS
- [ ] Web
- [X] Desktop (Go)

**Platform OS version**
This is platform-independent error

**Details to reproduce the issue**

1. Instantiate a `PlatformFile` without providing a path
2. Access the path

**Comment by miguelpruivo** [comment]
Hi, thank you, good catch.

**Comment by miguelpruivo** [comment]
Hi, thank you, good catch.

**Comment by miguelpruivo** [comment]
Fixed with 4.0.3.
